pegasus/sites.v3/code.org/public/afe/index.md.erb

```diff
@@ -9,11 +9,11 @@ theme: responsive
 
 Computer science is a foundational subject for a 21st century education. Students who study CS [perform better in other subjects, excel at problem solving, and are 17% more likely to attend college](https://medium.com/@codeorg/cs-helps-students-outperform-in-school-college-and-workplace-66dd64a69536). By offering CS to your students, you’re helping create new opportunities for them!
 
-Code.org has partnered with Amazon Future Engineer (AFE) to offer middle and high school teachers at underserved schools a growing number of resources and benefits at no cost!
+Code.org has partnered with Amazon Future Engineer to offer middle and high school teachers at underserved schools a growing number of resources and benefits at no cost!
 
-**The Amazon Future Engineers program is offered to schools with Title 1 status OR that have student enrollment of more than 40% underrepresented minority students and/or 40% students who qualify for free or reduced lunch.**
+**The Amazon Future Engineer program is offered to schools with Title 1 status OR that have student enrollment of more than 40% underrepresented minority students and/or 40% students who qualify for free or reduced lunch.**
 
-Scroll down to learn more about what Code.org offers and the benefits of participating through AFE, or if you’re ready, fill out the short form below to see if your school is eligible!
+Scroll down to learn more about what Code.org and Amazon Future Engineer each offer and the benefits of participating, or if you’re ready, fill out the short form below to see if your school is eligible!
```

Comment:
Remove acronym, and be inclusive in description of benefits.

Reply:
Slight wording tweak to make it more digestible: "Scroll down to learn more about what Code.org and Amazon Future Engineer have to offer and the benefits of participating..."